apps/aeprimop/src/aeprimop.erl

```diff
@@ -2157,12 +2169,12 @@ timed_contract_call(Type, Fun, CallData, CTVersion) ->
 
     {Call, State}.
 
-process_timed_contract_call(Time, Type, Call, CallData, State,
+process_timed_contract_call(Time, Type, Call, CallData, _State,
                             #{vm := CtVMVsn, abi := CtABIVsn}) ->
     ReturnType = aect_call:return_type(Call),
     GasUsed = aect_call:gas_used(Call),
     CallDataSize = byte_size(CallData),
-    CtPK = aect_call:contract_pubkey(Call),
+    % CtPK = aect_call:contract_pubkey(Call),
```

Comment:
Why do you keep that in here?

Reply:
I just fixed the compiler warnings that someone (you?) left behind... No great thought went into that process.